Pansynchro.Connectors.Snowflake/SnowflakeUploader.cs

```diff
@@ -80,12 +82,13 @@ private async Task UploadAsync(string filename, StreamDescription streamName)
                 var lFilename = Path.GetFileName(filename);
                 conn.Execute($"copy into {SnowflakeUploader.GetTargetName(streamName)} from '@{stageName}/{lFilename}' file_format = (type = avro) match_by_column_name = case_insensitive;");
                 await conn.CloseAsync();
-            } catch (Exception e) {
-                Console.WriteLine(e);
+            } catch (Exception e) {                
```

Comment:
For some strange reason, there's a lot of extra whitespace at the end of this line.